src/components/Person/Person.jsx

```diff
@@ -1 +1,37 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const MALE = 'm';
+
+  const partnerType = () => {
+    if (person.sex === MALE) {
+      return 'wife';
+    }
+
+    return 'husband';
+  };
```

Comment:
In should be const, no need in function
```suggestion
  const partnerType = person.sex === MALE ? 'wife' : 'husband';
```